pvlib/iotools/bsrn.py

```diff
@@ -0,0 +1,142 @@
+"""Functions to read data from the Baseline Surface Radiation Network (BSRN).
+.. codeauthor:: Adam R. Jensen<adam-r-j@hotmail.com>
+"""
+
+import pandas as pd
+import gzip
+
+COL_SPECS = [(0, 3), (4, 9), (10, 16), (16, 22), (22, 27), (27, 32), (32, 39),
+             (39, 45), (45, 50), (50, 55), (55, 64), (64, 70), (70, 75)]
+
+BSRN_COLUMNS = ['day', 'minute',
+                'ghi', 'ghi_std', 'ghi_min', 'ghi_max',
+                'dni', 'dni_std', 'dni_min', 'dni_max',
+                'empty', 'empty', 'empty', 'empty', 'empty',
+                'dhi', 'dhi_std', 'dhi_min', 'dhi_max',
+                'lwd', 'lwd_std', 'lwd_min', 'lwd_max',
+                'temp_air', 'relative_humidity', 'pressure']
+
+
+def read_bsrn(filename):
+    """
+    Read a BSRN station-to-archive file into a DataFrame.
+
+    The BSRN (Baseline Surface Radiation Network) is a world wide network
+    of high-quality solar radiation monitoring stations as described in [1]_.
+    The function only parses the basic measurements (LR0100), which include
+    global, diffuse, direct and downwelling long-wave radiation [2]_. Future
+    updates may include parsing of additional data and meta-data.
+
+    BSRN files are freely avaiable and can be accessed via FTP [3]_. Required

```

Comment:
```suggestion
    BSRN files are freely available and can be accessed via FTP [3]_. Required

```

Reply:
I accidentally suggested an extraneous newline here -- I tried to edit the suggestion in time, but looks like I wasn't quick enough.  It's small enough that I don't care if it gets fixed here.  Thanks @AdamRJensen for this contribution!